syntax = "proto3";

package node_rpc;

// The service definition.
service NodeRpc {
  rpc RpcMethod (Request) returns (Response) {}
}

// The request message
message Request {
  string input = 1;
}

// The response message
message Response {
  string output = 1;
  string status_message = 2;
  bool status = 3;
}